esp/esp/users/forms/user_profile.py

```diff
@@ -310,6 +310,14 @@ def clean(self):
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['rows'] = 8
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['cols'] = 45
 
+affiliationChoices = (
```

Comment:
Minor thing: this should be a constant (i.e. `ALL_CAPS`), and it probably makes more sense to put it in models.